install/kubernetes/quick-install.yaml

```diff
@@ -469,7 +469,7 @@ spec:
               name: cilium-config
               optional: true
         image: "docker.io/cilium/cilium:latest"
```

Comment:
This may be a bug in the helm chart. Still pointing to latest

Reply:
Fixed manually:

```
$ grep "latest" quick-install.yaml
$
```

Can file an issue to make sure the `quick-install.yaml` chart is updated when we run `make` as well so as to not block the release.